packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -54,21 +54,16 @@
 #include <Zoltan2_Parameters.hpp>
 #include <Zoltan2_Algorithm.hpp>
 #include <Zoltan2_IntegerRangeList.hpp>
-#include <Teuchos_StandardParameterEntryValidators.hpp>
-
+#include <Zoltan2_CoordinatePartitioningGraph.hpp>
+#include <Zoltan2_Util.hpp>
 #include <Tpetra_Distributor.hpp>
+#include <Teuchos_StandardParameterEntryValidators.hpp>
 #include <Teuchos_ParameterList.hpp>
-#include <Zoltan2_CoordinatePartitioningGraph.hpp>
+
 #include <new>          // ::operator new[]
```

Comment:
You don't need to include `<new>` just to use default `operator new`.

Reply:
Done